content/pages/guides/membership.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Membership Guide'
+description: 'Detailed information on Coding Train membership program!'
+date: '06/02/2023'
+---
+
+## Welcome!
+
+Thank you for considering supporting the Coding Train! We’ve recently made some changes to simplify the tiers and refocus on the community aspect.
+
+## What are the changes?
+
+The membership levels are now simplified to 2 tiers across all platforms: “Passenger” and “Conductor”.
+
+### Passengers $5 / month
+
+- Discord channels to chug along with The Coding Train team and other members
+- An annual mailing of the Coding Train sticker pack
+- Periodic “behind the scenes” live streams and Q&A session in the engine room
+
+### Conductors $10 or more / month
+
+- Everything in Passenger tier + mystery gift annually
+
+## What are the ways to join?
+
+You can support the Coding Train through any of the following platforms. Each may offer extra platform-specific perks (like emoji badges on YouTube), but everyone will get the rewards listed above!
+
+- [YouTube Membership](https://youtube.com/thecodingtrain/join)
+- [GitHub Sponsorshop](https://github.com/sponsors/shiffman)
+- [Patreon Support](https://www.patreon.com/codingtrain)
+- [Twitch Subscription](https://www.twitch.tv/subs/CodingTrainChooChoo): _twitch is presently not actively used but may be again in the future!_
+
+Physical rewards (stickers and mystery gift) will be sent once a year, all together. Annoucements and address collection will happen in the members Discord channels. Climb aboard to receive the first mailing this summer 2023!
+
+## What does this mean for existing members?
+
+We promise this is good news for you! You’ll still receive the benefits of the tiers you’re in, regardless of what you’ve received in the past, but now with more a consistent system to receive the rewards every year.
+
+If you're an existing member and missed any past rewards, please contact us at **[help@thecodingtrain.com](mailto:help@thecodingtrain.com)** to get back on track!
+
+## Why these changes?
+
+After trying various membership models, we found the train was breaking down and often late. The support truly keeps the engines running here but we haven’t been able to support the current membership system adequately. However, we think that one annual mailing for those who are members rather than piecemeal things is something we can manage! At its core, being a member is about supporting the community, so thank you for riding along and being a part of the community - Choo Choo!
```

Comment:
```suggestion
After trying various membership models, we found the merchandise delivery engine was breaking down and often late. Your support truly keeps the engines running here,  and we would like to do a better job of thanking you!  We think that one annual mailing rather than the current piecemeal approach is something we can manage. At its core, being a member is about supporting the Coding Train community, so we thank you for riding along - Choo Choo!
```

Reply:
@kchan2000 note these changes to this paragraph! We can mirror this in the instagram post, etc.